openmoc/thisown.i

```diff
@@ -90,12 +90,12 @@
 /* A Lattice owns the memory for each Universe it contains */
 %pythonappend Lattice::setUniverses %{
   # SWIG 3
-  if 'num_x' in locals():
-    universes = locals()['num_x']
-  elif 'args' in locals() and 'num_x' in locals()['args']:
-    universes = locals()['args']['num_x']
-  elif 'kwargs' in locals() and 'num_x' in locals()['kwargs']:
-    universes = locals()['kwargs']['num_x']
+  if 'num_y' in locals():
```

Comment:
If we decide to move to a streamlined 3D lattice universes input, make sure to update this